docker-compose.yaml

```diff
@@ -93,8 +93,15 @@ services:
       dockerfile: Dockerfile
     ports:
       - "3005:3005"
-    env_file:
-      - ./collaboration-service/.env
+    environment:
+      - PGHOST=postgres
+      - PGPORT=5432
+      - PGUSER=postgres
+      - PGPASSWORD=123
+      - PGDATABASE=dev
+      - PGSSLMODE=disable
+      - REST_PORT=3005
+      - USER_SERVICE_PORT=3000
```

Comment:
Uhh why are we hard coding the environment variables inside the compose

Reply:
it will be temporary until we move it to GKE secret and config maps during deployment